prefix.go

```diff
@@ -182,13 +182,28 @@ func (i *ipamer) AcquireChildPrefix(parentCidr string, length uint8) (*Prefix, e
 	var prefix *Prefix
 	return prefix, retryOnOptimisticLock(func() error {
 		var err error
-		prefix, err = i.acquireChildPrefixInternal(parentCidr, length)
+		prefix, err = i.acquireChildPrefixInternal(parentCidr, "", length)
+		return err
+	})
+}
+
+func (i *ipamer) AcquireSpecificChildPrefix(parentCidr, childCidr string) (*Prefix, error) {
+	var prefix *Prefix
+	return prefix, retryOnOptimisticLock(func() error {
+		var err error
+		cp, err := netaddr.ParseIPPrefix(childCidr)
```

Comment:
parsing of childCidr is done again inside the private func

Reply:
is it parsing it only in the exported function? I need to pass the length to the private func